i18n/zh/docusaurus-plugin-content-docs/current/architecture/edge/devicetwin.md

```diff
@@ -2,353 +2,327 @@
-   - The device state updation is initiated by publishing a message on the specified topic which is being subscribed by the eventbus component.
-   - The eventbus component sends the message to the device twin controller which forwards the message to the device module.
-   - The device module updates the state of the device as well as the last online time of the device in the database.
-   - The device module then sends the result of the device state update to the eventbus component and edgehub component through the communicate module of devicetwin. The eventbus component further publishes the result on the specified topic, while the
-   edgehub component sends the device status update to the cloud.
+ **dealDeviceStateUpdate**:  dealDeviceStateUpdate()处理遇到设备状态更新时要执行的操作。它应用了对设备属性的更改，例如数据库中属性的添加、属性更新和属性删除。它还将要发布的设备属性更新的结果发送到 EventBus 组件。
+
+
+   - 设备状态更新是通过在 eventbus 组件订阅的指定 topic 上发布消息来启动的。
+   - eventbus 组件将消息发送到 DeviceTwin Controller，控制器将消息转发到设备模块。
+   - 设备模块更新数据库中设备的状态以及设备的最后在线时间。
+   - 设备模块将设备状态更新的结果通过 devicetwin 的 communication 模块发送给 eventbus 组件和 edgehub 组件。 eventbus 组件进一步在指定 topic 上的发布结果，与此同时 edgehub 组件将设备状态更新发送到云端。
 
   ![Device State Update](/img/devicetwin/device-state-update.png)
 
 
-## Tables
+## 数据表
 
-DeviceTwin module creates three tables in the database, namely:
+DeviceTwin 模块在数据库中创建三个表，分别是：
 
-   - Device Table
-   - Device Attribute Table
-   - Device Twin Table
+   - Device Table：设备表
+   - Device Attribute Table：设备属性表
+   - Device Twin Table：设备孪生属性表
 
 
-### Device Table
+### 设备表
 
-Device table contains the data regarding the devices added to a particular edge node.
-The following are the columns present in the device table :
+设备表包含有关添加到特定边缘节点的设备的数据。以下是设备表中存在的列：
 
-|Column Name | Description |
+|列名 | 描述 |
 |---|---|
-| **ID** |  This field indicates the id assigned to the device |
-| **Name** | This field indicates the name of the device |
-| **Description** | This field indicates the description of the device |
-| **State** | This field indicates the state of the device |
-| **LastOnline** | This fields indicates when the device was last online |
+| **ID** |  该字段表示分配给设备的ID |
+| **Name** | 该字段表示设备的名称 |
+| **Description** | 该字段表示设备的描述信息 |
+| **State** | 该字段表示设备的状态 |
+| **LastOnline** | 该字段表示设备上次在线的时间 |
 
-**Operations Performed:**
+**可执行的操作：**
 
-The following are the operations that can be performed on this data:
+以下是可以对此数据执行的操作：
 
-- **Save Device**: Inserts a device in the device table
+- **保存设备**: 在设备表中插入设备
 
-- **Delete Device By ID**: Deletes a device by its ID from the device table
+- **按 ID 删除设备**: 按 ID 从设备表中删除设备
 
-- **Update Device Field**: Updates a single field in the device table
+- **更新设备单字段**: 更新设备表中的单个字段
 
-- **Update Device Fields**: Updates multiple fields in the device table
+- **更新设备多个字段**: 更新设备表中的多个字段
 
-- **Query Device**: Queries a device from the device table
+- **查询设备**: 从设备表中查询设备
 
-- **Query Device All**: Displays all the devices present in the device table
+- **查询全部设备**: 显示设备表中存在的所有设备
```

Comment:
Operation names do not need to be translated

Reply:
Thank you, the entire document has been checked and revised.